### PR TITLE
Added support to get Product data using bulk query 

### PR DESCRIPTION
**Pull request by Nishtha-Jain-1119**
Added system message types, job and template date for getting product metafields by tags for the given namespace.

closes: [#1189](https://git.hotwax.co/HC2/plugins/ofbiz-oms-usl/-/issues/1189)

### CODE REVIEW

File: data/ShopifySetupSeedData.xml [hunk 1]
```diff
@@ -491,4 +491,36 @@ under the License.
...
+            consumeServiceName="co.hotwax.shopify.system.ShopifySystemMessageServices.consume#BulkOperationResult"
+            receivePath="${contentRoot}/shopify/BulkProductMetaFieldByTagsQuery/BulkOperationResult-${systemMessageId}-${remoteMessageId}-${nowDate}.jsonl">
+        <parameters parameterName="consumeSmrId" parameterValue="" systemMessageRemoteId=""/>
+    </moqui.service.message.SystemMessageType>
+
+    <moqui.service.message.SystemMessageType systemMessageTypeId="SendBulkProductMetaFieldsQueryResult"
+            description="Send Bulk Product Metafields by Tags Query Result"
+            sendServiceName="co.hotwax.ofbiz.ProductServices.process#BulkProductMetaFieldByTagsQueryResult"
+            sendPath="${contentRoot}/shopify/BulkProductMetaFieldByTagsResult/BulkProductQueryResult-${systemMessageId}-${dateTime}.json">
+        <parameters parameterName="consumeSmrId" parameterValue="RemoteSftp" systemMessageRemoteId=""/>
```
Comment (mridulpathak) [inline data/ShopifySetupSeedData.xml:510]: Keep parameterValue here empty in seed data.

File: data/ShopifySetupSeedData.xml [hunk 2]
```diff
@@ -491,4 +491,36 @@ under the License.
...
+    </moqui.service.message.SystemMessageType>
+
+    <moqui.service.message.SystemMessageType systemMessageTypeId="ProductCategoryMemberFeed"
```
Comment (mridulpathak) [inline data/ShopifySetupSeedData.xml:513]: This seems out of place here, what is this SystemMessageType for and where is it's consume service implementation?

File: data/ShopifySetupSeedData.xml (hunk 1 above)
Comment (Nishtha-Jain-1119) [reply data/ShopifySetupSeedData.xml]: Made the required changes

File: data/ShopifySetupSeedData.xml (hunk 2 above)
Comment (Nishtha-Jain-1119) [reply data/ShopifySetupSeedData.xml:513]: This SystemMessageType is used to generate product category member feed by using the bulk operation result data, and then put it to SFTP for further processing.

The consume service implementation is in ofbiz-oms-usl.

https://git.hotwax.co/HC2/plugins/ofbiz-oms-usl/-/merge_requests/509

**Pull request merged**